<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoValueGeneratorSettingsWindow</class>
 <widget class="QDialog" name="AutoValueGeneratorSettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auto Value Generator Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="table_widget">
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Column Name</string>
      </property>
      <property name="toolTip">
       <string>Actual name of the column, use it as id in formula.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Data Type</string>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Type of the data, &lt;span style=&quot; font-weight:700;&quot;&gt;if you use formula then this data type and return type of the formula must be same.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Display Name</string>
      </property>
      <property name="toolTip">
       <string>Display name is used in different places to make it more designable</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fill Type</string>
      </property>
      <property name="toolTip">
       <string>Describe how to generate the value</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fill With</string>
      </property>
      <property name="toolTip">
       <string>This will be used to fill the column automatically</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="_ignore_label_2">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Automate and speed up adding rows. Automate fields using &lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; font-style:italic;&quot;&gt;auto increament&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; and &lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; font-style:italic;&quot;&gt;formula&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;. &lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; font-style:italic;&quot;&gt;Note :&lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-style:italic;&quot;&gt; Formula should not depend on the next columns it should only include previous columns.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_ignore_horizontal_layout_2">
     <item>
      <widget class="QLabel" name="lb_warning_icon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="_ignore_label">
       <property name="font">
        <font>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Warning: New Column can not be added till this window is opened.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_ignore_horizontal_layout">
     <item>
      <widget class="QPushButton" name="btn_attacher">
       <property name="text">
        <string>Attach To Tab</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="_ignore_horizontal_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>AutoValueGeneratorSettingsWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
